README.md

```diff
@@ -106,6 +106,7 @@ Represents a shopping basket
 | lineItems | <code>[ &#x27;Array&#x27; ].&lt;LineItem&gt;</code> | Line items contained in the basket |
 | [transactions] | <code>[ &#x27;Array&#x27; ].&lt;Transaction&gt;</code> | Transactions on the basket |
 | [discounts] | <code>[ &#x27;Array&#x27; ].&lt;Discount&gt;</code> | Discounts on the basket |
+| customerId | <code>string</code> &#124; <code>null</code> | Baskets Customers id |
```

Comment:
Other optional entities are enclosed with brackets `[]`, however, `customerId` is describing nullability in a new way. Is this correct?